### DISCUSSION TITLE
autofocus

### DISCUSSION THREAD

**Discussion by aejuice-github** [aidenlx/media-extended#350] [Q&A]
Is it possible to autofocus on note (place cursor) so I could start typing right away?

It's very annoying to place your cursor every few seconds.

Additionally, once you switch to the note tab, the shortcut for timestamp does not work anymore.

So now you have to switch back to the tab with video, press shortcut and now you have to do the same stuff again of placing cursor.

I saw a road map of adding some advance features but please could you do the basics? It would be just amazing to press a shortcut and type your note instead of juggling mouse and keep in mind where your cursor at the moment.

Thanks!

**Comment by aejuice-github** [comment]
Here is a demo https://drive.google.com/file/d/1QdaWq6OrqfQ1ZmWjIx5wodBN66weEX5x/view?usp=sharing

Doesn't it look painful to you?

**Comment by aejuice-github** [comment]
The podcast plugin for taking notes does not have similar issues, maybe you could take a look of how they did it?

**Comment by aidenlx** [comment]
Could you first describe your workflow in details, including the command you used? In general, most user would find using hotkeys to control player from note less distracting.
